account_payment_return/models/payment_return.py

```diff
@@ -231,6 +254,13 @@ class PaymentReturnLine(models.Model):
         string='Amount',
         help="Returned amount. Can be different from the move amount",
         digits=dp.get_precision('Account'))
+    expense_account = fields.Many2one(
```

Comment:
Put at journal level a default expense account and expense partner, and add an `_onchange_expense_amount` to bring this data to the fields.